edx-platform/pearson-vue-theme/lms/templates/courseware/course-navigation.html

```diff
@@ -35,6 +37,7 @@
 
 % if disable_tabs is UNDEFINED or not disable_tabs:
     <%
+    ALTERNAVITE_STRINGS = configuration_helpers.get_value('ALTERNAVITE_STRINGS', {})
```

Comment:
Is this necessary here? Are you going to use ALTERNATIVE_STRINGS? @luismorenolopera

Reply:
It will not be used, but I think it is better to leave it, I think it is good to have a configuration available instead of having to add it to via template to have it available again.

Let me know if you agree

Reply:
But they don't even know it's available so they won't use it, this was for a specific business case unrelated to this theme so it's not needed and it's better to remove it. @luismorenolopera